infra/terraform/modules/privatelink_endpoints/main.tf

```
@@ -0,0 +1,74 @@
+variable "region" {
+  type        = string
+  description = "The AWS region"
+}
+
+output "endpoints" {
+  value = {
+    # Allow secrets to be fetched from the private subnet
+    # SSM contain the Parameter Store, which stores secrets.
+    ssm = {
+      name        = "com.amazonaws.${var.region}.ssm"
+      type        = "Interface"
+      private_dns = false
+      phz_name    = "ssm.${var.region}.amazonaws.com"
+      alias       = [""]
+    }
+    # Allow Fargate Exec control messages
+    ssmmessages = {
+      name        = "com.amazonaws.${var.region}.ssmmessages"
+      type        = "Interface"
+      private_dns = false
+      phz_name    = "ssmmessages.${var.region}.amazonaws.com"
+      alias       = [""]
+    }
+    # Used for various Fargate related operations and Fargate Exec
+    ec2messages = {
+      name        = "com.amazonaws.${var.region}.ec2messages"
+      type        = "Interface"
+      private_dns = false
+      phz_name    = "ec2messages.${var.region}.amazonaws.com"
+      alias       = [""]
+    }
+    # Used for logging to CloudWatch
+    ec2messages = {
```

Is this supposed to be called `ec2logs`?
```suggestion
    ec2logs = {
```

---

Yes, thanks!